eoscostarica/eosio-dashboard
# Feat/endpoints bug pagination

### Endpoints bug pagination

### What does this PR do?

- Resolve #927
- Use Sets when endpoints are saved to avoid duplication
- Remove box and grid from endpointsList component
- Split State from endpointsList component
- Split EndpointTable component from endpointsList component

### Steps to test

1. Run the project locally
1. Go to endpoints page
1. Check the pagination

### Screenshots

![imagen](https://user-images.githubusercontent.com/66583677/192621499-04a7be22-c073-49a3-80c7-22ac5857a339.png)


#### CheckList
